tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1874,6 +1874,13 @@ def rgb_to_grayscale(images, name=None):
 
   Returns:
     The converted grayscale image(s).
+    
+  Usage Example:
+   ```python
+   import tensorflow as tf
```

Comment:
No need for importing tensorflow

Reply:
Ok, I will get rid of that. Thanks for the feedback.